docs/README.md

```diff
@@ -1,143 +1,581 @@
-| scrollOptions | `RenderScrollOptions` | -- | After the navigation, if the page should be scrolled to a specific position, or should stay still (use `false`)
-| replace | `boolean` | `undefined` | If it should call the replace function to navigate or not
-| preventRemount | `boolean` | `false` | If `true`, only the URL will change, but not the components :exclamation: **Use with caution!**
+**Example value:**
+`"store.home"`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { page } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+### `production`
+
+A _boolean_ value representing whether the app is in a production workspace or not.
+
+**Example value:**
+`false`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { production } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+### `query`
+
+An _object_ that stores the query string values in a key-value format.
+
+**Example value:**
+`{ "foo": "bar" }`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { query } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+### `renderMajor`
+
+A _number_ with the major version of Render Runtime.
+
+**Example value:**
+`8`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { renderMajor } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+### `rootPath`
+
+A _string_ with the root path of the store. It can be `undefined` if none is set.
+
+**Example value:**
+`/ar`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { rootPath } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+### `setQuery`
+
+A _function_ that can be called to set query string params.
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { setQuery } = useRuntime()
+
+  const handleClick = () => {
+    setQuery({ foo: 'bar' })
+  }
+
+  return <button>Set</button>
+}
 
+export default MyComponent
+```
+
+### `workspace`
+
+A _string_ with the current workspace name.
+
+**Example value:**
+`master`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { workspace } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+## Block (alias ExtensionPoint)
+
+A React component that is used to create Store Framework blocks that expects a specific block id.
```

Comment:
We could add a disclaimer here, telling people that they should use `Block` or `ExtensionPoint` if they expect to receive blocks via the `blocks` array.